src/routes/api/api-replay-failed-entities-from-data-depot.ts

```diff
@@ -0,0 +1,216 @@
+			};
+			const vulnPayload = await transformDependabotAlerts([dependabotAlertResponseItem], repository, subscription.jiraHost, logger, undefined);
+			logger.info(`Transformed to Jira vulnerability successfully repoId - ${repoId}, alert ${alertNumber}`);
+			if (vulnPayload.vulnerabilities?.length > 0) {
+				return vulnPayload.vulnerabilities[0];
+			}
+
+		}
+		logger.info(`Failed to fetch dependabot alert from GitHub repoId - ${repoId}, alert ${alertNumber}`);
+	}
+	return undefined;
+};
+
+const getCodeScanningVulnPayload = async (
+	gitHubInstallationClient: GitHubInstallationClient,
+	subscription: Subscription,
+	repoId: number,
+	alertNumber: number,
+	logger: Logger
+): Promise<JiraVulnerability | undefined> => {
+	const repoSyncState = await RepoSyncState.findByRepoId(subscription, repoId);
+	if (repoSyncState) {
+		const { data: codeScanningAlertResponseItem } = await gitHubInstallationClient.getCodeScanningAlert(repoSyncState.repoOwner, repoSyncState.repoName, alertNumber);
+		if (codeScanningAlertResponseItem) {
+			logger.info(`Fetched code scanning alert successfully from GitHub repoId - ${repoId}, alert ${alertNumber}`);
+			const repository: Repository = {
+				id: repoId,
+				full_name: repoSyncState.repoFullName,
+				name: repoSyncState.repoName,
+				owner: { login: repoSyncState.repoOwner },
+				html_url: repoSyncState.repoUrl,
+				updated_at: repoSyncState.updatedAt.toString()
+			};
+			logger.info(`Transformed to Jira vulnerability successfully repoId - ${repoId}, alert ${alertNumber}`);
+			const vulnPayload = await transformCodeScanningAlert([codeScanningAlertResponseItem], repository, subscription.jiraHost, logger, undefined);
+			if (vulnPayload.vulnerabilities?.length > 0) {
+				return vulnPayload.vulnerabilities[0];
+			}
+
+		}
+		logger.info(`Failed to fetch code scanning alert from GitHub repoId - ${repoId}, alert ${alertNumber}`);
+	}
+	return undefined;
+};
+
+const getSecretScanningVulnPayload = async (
+	gitHubInstallationClient: GitHubInstallationClient,
+	subscription: Subscription,
+	repoId: number,
+	alertNumber: number,
+	logger: Logger
+): Promise<JiraVulnerability | undefined> => {
+	const repoSyncState = await RepoSyncState.findByRepoId(subscription, repoId);
+	if (repoSyncState) {
+		const { data: secretScanningAlertResponseItem } = await gitHubInstallationClient.getSecretScanningAlert(alertNumber, repoSyncState.repoOwner, repoSyncState.repoName);
+		if (secretScanningAlertResponseItem) {
+			logger.info(`Fetched secret scanning alert successfully from GitHub repoId - ${repoId}, alert ${alertNumber}`);
+			const repository: Repository = {
+				id: repoId,
+				full_name: repoSyncState.repoFullName,
+				name: repoSyncState.repoName,
+				owner: { login: repoSyncState.repoOwner },
+				html_url: repoSyncState.repoUrl,
+				updated_at: repoSyncState.updatedAt.toString()
+			};
+			logger.info(`Transformed to Jira vulnerability successfully repoId - ${repoId}, alert ${alertNumber}`);
```

Comment:
Nit: Probably don't need both of these logs right? Could just keep this one and remove the fetched log